docs/articles/scaleout.md

```diff
@@ -0,0 +1,154 @@
+# Easy scaling of a zot cluster
+
+> :point_right: A cluster of zot replicas can easily be scaled with no repo-specific programming of the load balancer using:
+>
+> -   Stateless zot instances to simplify scale out
+> -   Shared remote storage
+> -   zot release v2.1.0 or later
+
+Beginning with zot release v2.1.0, a new "scale-out" architecture greatly reduces the configuration required when deploying large numbers of zot instances.  As before, multiple identical zot replicas run simultaneously using the same shared reliable storage, but with improved scale and performance in large deployments. 
+
```

Comment:
Scale-out is achieved by automatically sharding based on repository name so that each zot instance is responsible for a subset of repositories.

Reply:
In the cloud deployment case, the backend (for example S3) and metadata storage (for example dynamodb) can be scaled along with the zot instances.

Reply:
Revised